### PR TITLE
Changed logging configs, added logging format

### PR DESCRIPTION
**Pull request by VeronikaHh**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 
 ROOT_DIR = Path(__file__).parent
 SERVICE_NAME = "LIBRARY_MANAGEMENT_API_"
+DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S"
+LOGGING_FORMAT = "%(asctime)s %(levelname)s %(message)s"
+
 
 class SystemConfig(BaseSettings):
-    log_level: str = "INFO"
+    log_level = logging.INFO
```
Comment (taras-hrechukh) [inline config.py:13]: Just for future... you still need type annotation here -> `log_level: str = logging.INFO`

**Review by taras-hrechukh — APPROVED** [review]
Very good job!

File: config.py (hunk 1 above)
Comment (VeronikaHh) [reply config.py:13]: Sure, gonna change it now and then merge it. *One note this is not str but int
Знімок екрана 2024-12-24 о 14 19 28

Comment (taras-hrechukh) [reply config.py]: that's a good point, don't put `int` annotation there as you as a developer won't know th int code for specific logging level and you will put string instead

a bit tricky one and I see here two solutions:
1) you can ommit setting the type annotation in this place and just leave it as is
2) you need to set this default value to be str not int and put str to your type annotation

Comment (taras-hrechukh) [reply config.py]: here is the best solution 
`info_level_string = logging.getLevelName(logging.INFO)`

Comment (VeronikaHh) [reply config.py]: already merged)))) 
will change it in the next pull request, with the first solution(as it was previously).

Comment (VeronikaHh) [reply config.py]: Ok then before testing will create refactoring PR

**Pull request merged**